packages/react-instantsearch-dom-maps/src/Connector.js

```diff
@@ -1,10 +1,26 @@
-import { isEqual } from 'lodash';
 import { Component } from 'react';
 import { polyfill } from 'react-lifecycles-compat';
 import PropTypes from 'prop-types';
 import { connectGeoSearch } from 'react-instantsearch-dom';
 import { LatLngPropType, BoundingBoxPropType } from './propTypes';
 
+function isEqualPosition(a, b) {
+  if (a === b) {
+    return true;
+  }
+  return a.lat === b.lat && a.lng === b.lng;
+}
+
+function isEqualCurrentRefinement(a, b) {
```

Comment:
We have `fast-deep-equal` inside the package since it's already used in `core`. I would have leverage it rather than this two custom functions. The simpler, the better.